README.md

```diff
@@ -155,13 +155,25 @@ be able to see the line "You are logged in as ".
 Once Basic HTTP auth works test running the robot task from the CLI:
 
 ```
-php admin/tool/task/cli/schedule_task.php --execute='\tool_crawler\task\crawl_task'
+php admin/tool/task/cli/schedule_task.php --execute='\tool_crawler\task\crawl_manager'
```

Comment:
I think this is out of date, might need to rebase onto to the parallel task commits or merge it first

Reply:
Have rebased on master now, and the PR target branch is master instead of issue94